File: Data Structures & Algorithms/divide-array-into-equal-pairs/submission-0.cpp
class Solution {
public:
    bool divideArray(vector<int>& nums) {
        for(int i=0;i<nums.size();i++){
            int c=0;
            for(int j=0;j<nums.size();j++){
                if(nums[i]==nums[j]){
                    c++;
                }
            }
            if(c%2!=0){
                return false;
            }
        }
        return true;
    }
};

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        l=[]
        n=len(nums)/3
        for i in nums:
            count=0
            for j in nums:
                if i==j:
                    count+=1
            if count>n:
                l.append(i)
        ans=set(l)
        l2=list(ans)
        return l2
        